modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -10,3 +10,60 @@
 The Comments plugin provides the following {productname} commands.
 
 include::partial$commands/comments-cmds.adoc[]
+
+== Events
+
+include::partial$events/tinycomments-events.adoc[]
+
+=== CommentChange event
+
+This event triggers whenever:
+
+* A new comment is added (and who added it).
+* A comment is edited (and who edited it).
+* A reply to a comment is added (and who added the reply).
+* A comment is resolved (and who resolved the comment).
+* A comment is deleted (and who deleted the comment).
+
+==== Example: The CommentChange event
+
+include::partial$misc/admon-requires-6.1v.adoc[]
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinycomments',
+  toolbar: 'addcomment showcomments',
+  init_instance_callback: (editor) => {
+    editor.on('CommentChange', (evt) => {
+      console.log(evt.getEventLog());
+    });
+  }
+});
+----
+
+== APIs
+
+The {pluginname} plugin provides the following API.
+
+[[getEventLog]]
+=== getEventLog()
+
+include::partial$misc/admon-requires-6.1v.adoc[]
+
+Returns a log of all `CommentChange` events.
```

Comment:
@MitchCTiny as I said in my previous comment this isn't correct... it's not a log of `CommentChange` events, the two aren't the same thing. A `CommentChange` just means something has changed, it doesn't have the information. The `getEventLog()` API returns that explicit information about the comment state for the change.